src/logEntryProcessing.ts

```diff
@@ -35,7 +36,7 @@ function parseReportField(rawField: string): [string, any] {
 function lambdaRequestLogData(line: string): StructuredLogData | undefined {
     if (isRequestLogEntry(line)) {
         const eventName = line.substr(0, line.indexOf(' '));
-        const requestId = fieldValue(line, 'RequestId:', 36);
+        const requestId = fieldValue(line, 'RequestId', 36);
```

Comment:
To my mind, the `:` is not part of the field name, so I've removed these and coped with them in the function instead.